samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,142 @@
+This facilitates the reading of responses or analyzing of events related to the nRF9160 modem.
+The commands can be initiated from a terminal or the `LTE Link Monitor`_, which is an application implemented as part of `nRF Connect for Desktop`_.
+
+For more information on the AT commands, see the `AT Commands Reference Guide`_.
+
+Also see :ref:`modem_info_readme` for information about how an LTE application can interact with the modem.
+
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/at_client`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Connecting to the nRF9160 DK board with LTE Link Monitor
+========================================================
+
+To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
+
+#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
+#. Connect the nRF9160 DK board to the PC with USB.
+#. Power on the nRF9160 DK board.
+#. Click :command:`Select Device` and select the particular board device entry from the drop-down list in the LTE Link Monitor.
+#. Observe that initially the command :command:`AT+CFUN?` is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
+#. Observe that the LTE Link Monitor terminal display also shows :command:`AT+CFUN=1` indicating that the modem has changed to the normal mode.
+#. Perform a reset of the nRF9160 DK board by pressing the reset button, which will reboot the board and start the AT Client sample.
+
+
+Testing
+=======
+
+After programming the sample to your board, test the sample by performing the following steps:
+
+#. `Connect to the nRF9160 development board with LTE Link Monitor <Connecting to the nRF9160 DK board with LTE Link Monitor_>`_.
+#. Run the following commands from the LTE Link Monitor terminal:
+
+   * Enter the command :command:`AT+CFUN?`.
+
+     This command connects to an LTE network and triggers the command :command:`AT+CFUN=1` which will change the modem state to the normal mode.
+
+   * Enter the command :command:`AT+CFUN?` into the LTE Link Monitor terminal again.
+
+     The UART/Modem/UICC/LTE/PDN indicators in the LTE Link Monitor side panel turn green.
+     This command also automatically launches a series of commands like:
+
+     * :command:`AT+CGSN=1` which displays the product serial idenitification number(IMEI).
+     * :command:`AT+CGMI`   which displays the manufacturer name.
+     * :command:`AT+CGMM`   which displays the model identification name.
+     * :command:`AT+CGMR`   which displays the revision identification.
+     * :command:`AT+CEMODE` which displays the current mode of operation.
+
+   * Enter the command :command:`AT+CLAC`.
+
+     This command displays a list of all the available AT commands.
```

Comment:
If we promote the CLAC command, we should also take care to document them all in the AT commands documentation.
I'm not sure how the situation is now, but at the end of summer, there were ~20 undocumented commands.

Reply:
The AT+CLAC command is documented here: https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/clac.html

Reply:
The problem is not the CLAC command, but that in the list of supported commands, some of them are not documented. Running CLAC, and searching for the commands in the documentation give no results for the following commands at the moment:
CGCMOD
CGTFT
XDRX
XNTF
CSQ
VZWRSRP
XINACTTMR
XSETSOCKOPT
XTRACE
XDEVENV
VZWAPNE
VZWAPNTIME
XLABAPP
XOPCONF
XFOTASIZE
XUSIMLCK
XSMSFALLBACK
XCPSMS

Reply:
I think then we can remove this command from the testing section. Please suggest a command that is a good candidate to be highlighted here as a replacement of AT+CLAC .

Reply:
That sounds more like hiding the issue under a rug, as all the AT commands should be documented anyways. Not sure if a replacement is necessary, but AT%CMNG=1 might be a good candidate.

Reply:
I've opened an issue for updating the AT commands reference (TECHDOC-1559). It's definitely a good point, but nothing we can fix in this PR. ;-)